assets/src/edit-story/components/panels/design/preset/header.js

```diff
@@ -48,8 +48,8 @@ const buttonCSS = css`
 const AddPresetButton = styled.button`
   ${buttonCSS}
   svg {
-    width: 26px;
-    height: 28px;
+    width: 15px;
+    height: 15px;
   }
```

Comment:
Couldn't this be moved to button css? It should apply to the edit button too, but with just be irrelevant in case of edit-mode.